boards/launchxl/README.md

```diff
@@ -0,0 +1,52 @@
+## Platform specific instructions
+
+### Flashing
```

Comment:
I think this README is missing a section. It should start with a little overview of the board, a link to buy it and its datasheet, and a picture.